includes/admin/views/omise-page-settings.php

```diff
@@ -139,6 +139,7 @@
 										new Omise_Payment_Creditcard,
 										new Omise_Payment_Installment,
 										new Omise_Payment_Internetbanking,
```

Comment:
Here again, feels like this code could be simplified by having a general list of available payment methods somewhere. Then you could just update the list instead of updating this code

Reply:
@jonrandy how do you feel about this approach? 
https://github.com/omise/omise-woocommerce/pull/153/files